tests/fixtures/noqa/noqa_pre38.py

```diff
@@ -33,7 +33,7 @@ def decorated():
 
 def wrong_comprehension1():
     return [
-        node for node in 'ab' if node != 'a' if node != 'b'  # noqa: WPS307
+        node for node in 'ab' if node != 'a' if node != 'b' # noqa: WPS307
```

Comment:
Let's put 2 spaces before comment:


```suggestion
        node for node in 'ab' if node != 'a' if node != 'b'  # noqa: WPS307
```

Reply:
Changed